src/App.js

```diff
@@ -7,6 +7,7 @@ import { Config } from './libs/config';
 import DuosFooter from './components/DuosFooter';
 import DuosHeader from './components/DuosHeader';
 import {useHistory} from 'react-router-dom';
+import loadingImage from './images/loading-indicator.svg';
```

Comment:
Switching over to imports will allow Webpack to include images in its bundle and adjust internal references as needed.